samples/tfm_integration/psa_firmware/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+
+# Override the binary used by qemu
+set(QEMU_KERNEL_OPTION "-device;loader,file=${CMAKE_BINARY_DIR}/tfm_merged.hex")
```

Comment:
is this still needed when boards themselves can define this. like here:
https://github.com/zephyrproject-rtos/zephyr/blob/67295be71ec389b7a52139facf8691552d92d4d6/boards/arm/mps2_an521/board.cmake#L15-L20

Reply:
Shouldn't be needed at all now.